docs/src/content/docs/guides/create-interactive-grant.mdx

```diff
@@ -31,56 +32,62 @@ Although grants for creating incoming payments and quotes are non-interactive by
     Continuation Request
   </LinkOut>
 
-## Steps
+## Create an interactive grant
+
+<Tabs>
+<TabItem label='TypeScript'>
+<Ts />
+
+#### Get started
 
-### 1. Specify a redirect start mode
+<ChunkedSnippet
+  source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/grant/grant-outgoing-payment.ts'
+  chunk='1'
+/>
 
-Indicate your client is capable of directing the end-user to a URI to start an interaction.
+<ChunkedSnippet
+  source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/grant/grant-outgoing-payment.ts'
+  chunk='2'
+/>
 
-1. Add the `interact` object to your [grant request](/apis/auth-server/operations/post-request/#request-body).
-2. Specify `redirect` as the `start` mode. Open Payments only supports `redirect`.
+<ChunkedSnippet
+  source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/grant/grant-outgoing-payment.ts'
+  chunk='3'
+/>
 
-The URI will be provided by the authorization server in its response.
+#### Request an interactive grant
 
-```ts
-interact: {
-    start: ["redirect"],
-},
-```
+Indicate your client is capable of directing the end-user to a URI to start an interaction. We’ll use an outgoing payment grant as an example.
 
-### 2. Specify a redirect finish mode
+1. Add the `interact` object to your grant request.
+2. Specify `redirect` as the `start` mode. Open Payments only supports `redirect`. The redirect URI will be provided by the authorization server in its response.
```

Comment:
```suggestion
2. Specify `redirect` as the `start` mode. Open Payments only supports `redirect` today. The redirect URI will be provided by the authorization server in its response.
```
to indicate that more is coming?

Reply:
This note (OP only supports redirect) appears a few times in this doc, so I'll change each appearance to include "at this time".